src/Dev/SapphireTest.php

```diff
@@ -70,6 +72,16 @@ class SapphireTest extends PHPUnit_Framework_TestCase
      */
     protected $fixtureFactory;
 
+    protected $backupGlobalsBlacklist = array(
+        'injector',
```

Comment:
I'm a bit confused about these. Are all top level variables automatically global? We use `$injector` not `global $injector` in our code. I had a look at the phpunit code, and it seems to piggy-back off `$GLOBALS` which doesn't seem to include these variables.